website/docs/r/cloud_backup_snapshot_restore_job.html.markdown

```diff
@@ -88,6 +92,8 @@ description: |-
   }
 ```
 
+
+
 ## Argument Reference
```

Comment:
We need to also better explain below that if * `delivery_type_config.pointInTime` is set to true then one must also use one of the **pointInTime** optional values.  I think that's what tripped up the reporter in the ticket INTMDB-370.